### ISSUE TITLE
torch.nn.functional.cross_entropy function missing

### ISSUE THREAD

**Issue by xhuan8** [dotnet/TorchSharp#786]
Hi,

The function torch.nn.functional.cross_entropy is not implemented, only cross_entropy_loss provided, so I have to create Loss with cross_entropy_loss first then calculate cross entropy?

**Comment by GeorgeS2019** [comment]
- [torch.nn.functional.cross_entropy](https://pytorch.org/docs/stable/generated/torch.nn.functional.cross_entropy.html?highlight=cross%20entropy#torch.nn.functional.cross_entropy)
> Python function, in torch.nn.functional.cross_entropy

- [torch.nn.CrossEntropyLoss](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html?highlight=cross%20entropy#torch.nn.CrossEntropyLoss)

> Python class, in CrossEntropyLoss

**Comment by xhuan8** [comment]
@GeorgeS2019 I mean (torch.nn.functional.cross_entropy) not implemented in TorchSharp

**Comment by NiklasGustafsson** [comment]
@xhuan8 -- see line 400 in Losses.cs. If you are not using version 0.98.0, please upgrade.

https://github.com/dotnet/TorchSharp/blob/97fa3df7fc62706bd19c332273d07f8f29eb7b7f/src/TorchSharp/NN/Losses.cs#L400

**Comment by xhuan8** [comment]
thanks
